server/cache.js

```diff
@@ -17,6 +18,11 @@ module.exports = Cache;
 
 function Cache(ecs){
   Emitter.call(this);
+  this.tasks = LRU({
+    max: 1000000,
+    maxAge: ms('1d'),
```

Comment:
This means if we change an env var, we won't see it for a day, right? If so, this will inturuppt the @segmentio/site team's workflow quite a bit :/